Maestro/subscriptions.json

```diff
@@ -310,7 +317,14 @@
       "actionArguments": {
         "vsoBuildParameters": {
           "ScriptFileName": "update-dependencies\\update-dependencies.ps1",
-          "Arguments": "-EnvVars 'GITHUB_USER=dotnet-bot','GITHUB_EMAIL=dotnet-bot@microsoft.com','GITHUB_PASSWORD=`$(`$Secrets[`'DotNetBotGitHubPassword`'])','CLI_BRANCH=rel/1.0.0-preview2.1'"
+          "Arguments": [
+            "-EnvVars",
+            "'GITHUB_USER=dotnet-bot',",
+            "'GITHUB_EMAIL=dotnet-bot@microsoft.com',",
+            "'GITHUB_PASSWORD=`$(`$Secrets[`'DotNetBotGitHubPassword`'])',",
+            "'CLI_BRANCH=rel/1.0.0-preview2.1',",
+            "'CLI_RELEASE_MONIKER=preview2.1'"
```

Comment:
Should this be `preview2-1`?  The CLI artifacts use that format, and that's what the Dockerfiles use e.g. at https://github.com/dotnet/dotnet-docker-nightly/blob/master/1.1/debian/sdk/projectjson/Dockerfile#L20.